regression/scripts/B737/mission_B737.py

```diff
@@ -157,7 +157,7 @@ def base_analysis(vehicle):
 
     # ------------------------------------------------------------------
```

Comment:
Results are different because of changes to compressibility and induced drag in Anil's updated aero scripts. These are big differences so we might want to discuss them (for example L/D goes from about 16.5 to 18)